creator-cards/vincent-baaij.json

```diff
@@ -10,14 +10,19 @@
     }
   ],
   "feeds": [{
-    "type": "dotnet",
+    "type": "LinkTagRssType",
```

Comment:
dotnet or personal are the only options. LinkTagRssType is from the old format

Reply:
Oops. Reverted in next commit.

How long would it normally take to how up on the Creator list/page? Just curious

Reply:
You mean like to get your newest item listed? About 2 hours.

Reply:
No, I mean like in the image and on 'browse all creators' page![Screenshot_20230422_152249.jpg](https://user-images.githubusercontent.com/1761079/233787728-9b658f7e-5a51-46ad-a8ef-ae7d0d564e5b.jpg)